src/cmd/client/query-storage-deal.js

```diff
@@ -0,0 +1,24 @@
+const toUri = require('../../lib/multiaddr-to-uri')
+const { ok } = require('../../lib/fetch')
+const toCamel = require('../../lib/to-camel')
+
+module.exports = (fetch, config) => {
+  return async (cid, options) => {
+    options = options || {}
+
+    const url = `${toUri(config.apiAddr)}/api/client/query-storage-deal?arg=${encodeURIComponent(cid)}`
+    const res = await ok(fetch(url, { signal: options.signal }))
+
+    const storageDeal = toCamel(await res.json())
+
+    if (storageDeal.proposalCid) {
+      storageDeal.proposalCid = storageDeal.proposalCid['/']
+    }
+
+    if (storageDeal.proofInfo) {
+      storageDeal.proofInfo = toCamel(storageDeal.proofInfo)
+    }
```

Comment:
Need to fix the CID here too!

```suggestion
    }
    
    if (storageDeal.proofInfo && storageDeal.proofInfo.commitmentMessage) {
      storageDeal.proofInfo.commitmentMessage = storageDeal.proofInfo.commitmentMessage['/']
    }
```

Reply:
Added it as part of the commit.